vutran/dext
# chore(*): prettify and upgrade eslint

- adds `prettier`
- updates `eslint`
- uses `prop-types`

---

Now, I understand that this PR can be really noisy with 2.8K additions, but it's mostly a scroll through PR, except take note of these changes

- package.json https://github.com/DextApp/dext/pull/184/files#diff-50
- extended eslintrc https://github.com/DextApp/dext/pull/184/files#diff-1

--- 

next up (another PR)

- upgrade babel
- upgrade webpack

---

QA

check out this branch, and try out the following

- `yarn install && yarn lint && yarn build` (additional run a prod build)
 - `yarn dev`

Running the commands above, and dext is still up and running (locally at least 😉 )
